packages/otel/src/instrumentations/vercel-handler.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  TracerProvider,
+  MeterProvider,
+  propagation,
+  context,
+} from "@opentelemetry/api";
+import type {
+  Instrumentation,
+  InstrumentationConfig,
+} from "@opentelemetry/instrumentation";
+import { Server } from "node:http";
+
+export interface VercelHandlerInstrumentationConfig
+  extends InstrumentationConfig {}
+
+export class VercelHandlerInstrumentation implements Instrumentation {
+  instrumentationName = "@vercel/otel/handlers";
```

Comment:
Nit, to be more inline with the class name/file name (also it's more clear what it is):
```suggestion
  instrumentationName = "@vercel/otel/vercel-handler";
```